framework/src/main/java/org/checkerframework/framework/type/HashcodeAtmVisitor.java

```diff
@@ -31,8 +31,10 @@ protected Integer defaultAction(AnnotatedTypeMirror type, Void v) {
         if (type == null) {
```

Comment:
Two ATMs have the same hash code if they have the same top-level annotations and the same underlying type.  This means that `Map<String, Object>` and `Map<@Nullable String, @Nullable Object>` have the same hash code.

The above fact is implied by the class documentation, but I suggest stating it explicitly there.

Reply:
This isn't true.  `HashcodeAtmVisitor` scans the whole type and sums the hash code for each AnnotatedTypeMirror in the AnnotatedTypeMirror.  I updated the Class Javadoc to make this clearer.